packages/api-key/src/services/api-key-module-service.ts

```diff
@@ -359,7 +441,10 @@ export default class ApiKeyModuleService<TEntity extends ApiKey = ApiKey>
     const dbSecretKeys = await this.apiKeyService_.list(
       {
         type: ApiKeyType.SECRET,
-        revoked_at: null,
+        $or: [
+          { revoked_at: { $eq: null } },
+          { revoked_at: { $gt: new Date() } },
+        ],
       },
       {},
```

Comment:
todo: same comment for auto pagnation

Reply:
in fact everywhere you are expecting to get all tokens and not the default 15 limit